### ISSUE TITLE
browsers not reloaded when using proxy

### ISSUE THREAD

**Issue by iainhouston** [BrowserSync/recipes#5]
-  browsers are reloaded on restart  on port 3000 OK
-  files are watched and sass compiled OK
-  css file tree written to correct destination OK 
-  localhost:3001 shows browsers connected OK
- **BUT** completion of sass task does not result in browsers being reloaded 
- reloadDelay up to 10 secs no effect
- tunnel (and other options) employed to no positive effect
-  `sandbox.home:8888` is a MAMP host on the local machine

Am I not telling browsersync something by any chance? Or have I told it something incorrect?

```js
    var gulp        = require('gulp'),
        sass        = require('gulp-sass'),
        browserSync = require('browser-sync').create();
    var reload      = browserSync.reload;

    var sassSources     = ['sass/**/*.scss'];

    gulp.task('sass', function () {
      return gulp.src(sassSources)
        .pipe(sass({   
            includePaths: ['bower_components/compass-mixins/lib']}
        ))
        .pipe(gulp.dest('./css'))
        .pipe(reload({stream: true}))
    })


    gulp.task('watch-server', ['sass'], function() {
        browserSync.init( {
            // browsersync with a MAMP php server 
            // Note: BS still serves on port 3000
            proxy: "sandbox.home:8888",
            reloadOnRestart: true
        });
        gulp.watch(sassSources,     ['sass']);
    });

    gulp.task('default', ['watch-server']);
```

**Comment by shakyShane** [comment]
if the resulting file, for example, was `main.css`, does the HTML of the page you are viewing contain?

```html
<link rel="stylesheet" href="main.css" />
```

it sounds like everything is working correctly apart from the final part (Browsersync finding the correct `link` element in the Dom)

**Comment by iainhouston** [comment]
Thanks @shakyShane for the pointer to the solution. All working perfectly now.  

TL;DR Works great with Drupal Omega subtheme layouts.

The problem was that Drupal was generating `@import` statements for each of the css files. I enabled the Drupal `link_css` module so that `link` statements are now generated.

The `sass` task above produces a *directory* of `css` files arranged in a tree - several reasons including  the 'layout' functionality of the Drupal Omega theme and some vendor libraries etc.  - and whereas`gulp-concat` into a single `css` file might work in some situatons, it is not ok here  without significant refactoring, plus, in production, Drupal itself will aggregate and compress `css`.  

Nevertheless, browserSync - magically to me! - picks up the changes in the tree and injects  into the browser instantly. Great!

**Comment by shakyShane** [comment]
@iainhouston great news :)

Feel free to [vote for us](https://thenetawards.com/vote/open-source/browsersync/) if you're happy with Browsersync & the support :)

**Comment by iainhouston** [comment]
I did already, you can be sure.
